playbooks/roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+
+- name: Install dependencies for using Google Cloud Storage
+  apt: name={{ item }}
+  with_items:
+    - gcc
+    - python-dev
+    - python-setuptools
+    - libffi-dev
+    - libssl-dev
+  when: BACKUPS_PROVIDER == 'gs'
+  tags: ['backups']
+
+- name: Install boto plugin for Google Cloud Storage
+  pip: name={{ item.name }} version={{ item.version }}
+  with_items:
+    - {name: gcs-oauth2-boto-plugin, version: 1.9}
+    - {name: oauth2client, version: 1.5.2}
+    - {name: pyopenssl, version: 0.15.1}
```

Comment:
Probably not, just because this backup process only adds a script and creates a cron job. If it were a standalone Python package, then a requirements file might be a better choice.